src/server/save_and_load.R

```diff
@@ -130,6 +144,28 @@ handleLoad <- function(input, workingSet, surveyAndProgramData, spectrumFilesSta
                 modelRunState$simul <- NULL
                 modelRunState$state <- "not_run"
 
+                outputsPath <- "model_outputs/par.rds"
```

Comment:
Nice to have: hash file and check hash on load to see if user's have fiddled with it